users/migrations/0001_initial.py

```diff
@@ -0,0 +1,47 @@
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('password', models.CharField(max_length=128, verbose_name='password')),
+                ('last_login', models.DateTimeField(blank=True, null=True, verbose_name='last login')),
+                ('is_superuser', models.BooleanField(default=False, help_text='Designates that this user has all permissions without explicitly assigning them.', verbose_name='superuser status')),
+                ('email', models.EmailField(max_length=255, unique=True, verbose_name='email address')),
+                ('first_name', models.CharField(blank=True, max_length=30, verbose_name='first name')),
+                ('last_name', models.CharField(blank=True, max_length=30, verbose_name='last name')),
+                ('is_staff', models.BooleanField(default=False, help_text='Designates whether the user can log into this admin site.', verbose_name='staff status')),
+                ('is_active', models.BooleanField(default=True, help_text='Designates whether this user should be treated as active. Unselect this instead of deleting accounts.', verbose_name='active')),
+                ('date_joined', models.DateTimeField(auto_now_add=True, verbose_name='date joined')),
+                ('date_of_birth', models.DateField(blank=True, null=True, verbose_name='date of birth')),
+                ('updated_at', models.DateTimeField(auto_now=True, verbose_name='updated at')),
+                ('phone_number', models.CharField(blank=True, max_length=17, null=True, validators=[django.core.validators.RegexValidator(message="Phone number must be entered in the format: '999999999'. Up to 15 digits allowed.", regex='^\\d{9,15}$')])),
+                ('country', models.CharField(choices=[('Poland', 'Poland'), ('United States', 'United States'), ('United Kingdom', 'United Kingdom'), ('Germany', 'Germany'), ('France', 'France')], default='Poland', max_length=20)),
+                ('user_type', models.CharField(choices=[('Employee', 'Employee'), ('Manager', 'Manager'), ('Admin', 'Admin')], default='Employee', max_length=20)),
+                ('groups', models.ManyToManyField(blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of their groups.', related_name='user_set', related_query_name='user', to='auth.Group', verbose_name='groups')),
+                ('user_permissions', models.ManyToManyField(blank=True, help_text='Specific permissions for this user.', related_name='user_set', related_query_name='user', to='auth.Permission', verbose_name='user permissions')),
+            ],
+            options={
+                'verbose_name': 'user',
+                'verbose_name_plural': 'users',
+                'ordering': ('id',),
+            },
+            managers=[
+                ('objects', users.models.CustomUserManager()),
+            ],
+        ),
+    ]
```

Comment:
Did you regenerate migration file after all review changes?

Reply:
No, I thought i should do it during merge, but I will do it now with another fixup commit

Reply:
Done